src/contracts/Translation/Translation.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\Test\Core\Translation;
+
+use JMS\TranslationBundle\Translation\Comparison\ChangeSet;
+use JMS\TranslationBundle\Translation\ConfigFactory;
+use JMS\TranslationBundle\Translation\Updater;
+use Twig\Environment;
+use Twig\TwigFilter;
+use Twig\TwigFunction;
+
+final class Translation
```

Comment:
How about `TranslationService`, so it doesn't look like a `Value`?

Reply:
+1 given this line https://github.com/ibexa/test-core/pull/4/files#diff-91a06b220f2c897d192c7aa0455b5b1e8f7f547450ae0835d8af052567ed79c8R64.